services/web/pkg/config/config.go

```diff
@@ -33,6 +33,7 @@ type Config struct {
 type Asset struct {
 	DeprecatedPath string `yaml:"path" env:"WEB_ASSET_PATH" desc:"Serve ownCloud Web assets from a path on the filesystem instead of the builtin assets." introductionVersion:"pre5.0" deprecationVersion:"5.1.0" removalVersion:"6.0.0" deprecationInfo:"The WEB_ASSET_PATH is deprecated and will be removed in the future." deprecationReplacement:"Use WEB_ASSET_CORE_PATH instead."`
 	CorePath       string `yaml:"core_path" env:"WEB_ASSET_CORE_PATH" desc:"Serve ownCloud Web assets from a path on the filesystem instead of the builtin assets." introductionVersion:"5.1"`
+	ThemesPath     string `yaml:"themes_path" env:"WEB_ASSET_THEMES_PATH" desc:"Serve ownCloud Web themes from a path on the filesystem instead of the builtin assets." introductionVersion:"5.2"`
```

Comment:
Should we add something like:
`If not defined, the root directory derives from $OCIS_BASE_DATA_PATH:/<whatever-path>.`
which is a commonly used text component we have in many services/envvar descriptions, see a list in [admin docs](https://doc.owncloud.com/ocis/next/deployment/general/general-info.html#base-data-directory).

Reply:
the theme pickup location logic is pretty complex behind the scenes, if would love to not have that detail in here (we also have it not for the corePath).

in my opinion `Serve ownCloud Web themes from a path on the filesystem instead of the builtin assets.` says it all, specially if the user DOES NOT provide a WEB_ASSET_THEMES_PATH env, then he should not be interested what happens behind the scenes.

ok for you?

Reply:
No, this is the wrong approach, but lets have a chat to clarify tomorrow 😄

Reply:
@mmattel I would not go into customizing in the official docs. Customizing is IMHO always a "cherry on the cake".